vendor.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/golang/gddo v0.0.0-20190904175337-72a348e765d2
 	github.com/golang/protobuf v1.5.3
 	github.com/google/go-cmp v0.5.9
+	github.com/google/gopacket v1.1.19
```

Comment:
Same for this commit (needs to be squashed with the commit that starts using it)